### PR TITLE
Add Cloud Healthcare API to Kokoro testing pipeline

### PR DESCRIPTION
**Pull request by noerog**
Part of b/154638640

### CODE REVIEW

File: .kokoro/healthcare/datasets.cfg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Set the folder in which the tests are run
+env_vars: {
+    key: "PROJECT"
+    value: "healthcare/datasets"
+}
+
+# Tell the trampoline which build file to use.
+env_vars: {
```
Comment (fhinkel) [inline .kokoro/healthcare/datasets.cfg:10]: can we move this section to common.cfg because it's the same for all the healthcare samples?

**Comment by fhinkel** [comment]
LGTM once the new tests show up on CI and pass.

File: .kokoro/healthcare/datasets.cfg (hunk 1 above)
Comment (noerog) [reply .kokoro/healthcare/datasets.cfg]: Done.

**Comment by noerog** [comment]
@fhinkel I'm temporarily skipping the IAM tests which gets everything to pass, but I don't understand why the tests can't even recognize the healthcare/hl7v2 directory: https://source.cloud.google.com/results/invocations/78dbc059-289b-4e7c-aef0-4f5ef6a83f0c/targets/cloud-devrel%2Fnodejs-docs-samples%2Fpresubmit%2Fhealthcare%2Fhl7v2/log

Do you have any idea why?

**Comment by fhinkel** [comment]
```
allowed_env_vars: {
  key: "PROJECT"
  value: "([a-z]|/|-|_)+$"
}
```

Looks like the numbers in the project name are the problem. I can change the rule to allow numbers, too.

**Comment by noerog** [comment]
> ```
> allowed_env_vars: {
>   key: "PROJECT"
>   value: "([a-z]|/|-|_)+$"
> }
> ```
> 
> Looks like the numbers in the project name are the problem. I can change the rule to allow numbers, too.

Yes please if you could. This has to stay as "hl7v2".

**Comment by fhinkel** [comment]
Fixed, let's run the tests again.

**Comment by noerog** [comment]
All healthcare/ tests are now passing.

**Comment by fhinkel** [comment]
Awesome

**Comment by fhinkel** [comment]
Now we just need to unskip the IAM tests.

**Comment by noerog** [comment]
IAM tests are still failing. Still investigating why.

**Comment by fhinkel** [comment]
> IAM tests are still failing. Still investigating why.

Thanks!

**Comment by fhinkel** [comment]
`Error: Permission healthcare.datasets.setIamPolicy denied on resource projects/nodejs-docs-samples-tests/locations/us-central1/datasets/dataset_4759ee5c_43c8_4c86_a013_beba8ebf31ba (or it may be malformed or not exist)`

I don't see that dataset in console.cloud.google.com. Is it deleted by another test?

**Comment by noerog** [comment]
> `Error: Permission healthcare.datasets.setIamPolicy denied on resource projects/nodejs-docs-samples-tests/locations/us-central1/datasets/dataset_4759ee5c_43c8_4c86_a013_beba8ebf31ba (or it may be malformed or not exist)`
> 
> I don't see that dataset in console.cloud.google.com. Is it deleted by another test?

Yes, the dataset is deleted in https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/healthcare/datasets/system-test/datasets.test.js#L101.

**Comment by fhinkel** [comment]
Are the tests maybe executed in an order such that the dataset is deleted before the test tries to use it?

**Comment by noerog** [comment]
> Are the tests maybe executed in an order such that the dataset is deleted before the test tries to use it?

The tests are executed synchronously and deletion happens at the end. I pushed a commit that had the IAM tests run earlier (right after dataset creation), and the issue still appears.

**Comment by fhinkel** [comment]
@noerog , I checked the logs and changed the permission for the right service account. `healthcare/datasets` is passing now. 

The other healthcare tests are still failing: 

```
Error: Permission healthcare.fhirStores.setIamPolicy denied on resource projects/nodejs-docs-samples-tests/locations/us-central1/datasets/nodejs_docs_samples_test_8be4a9bd_81c7_41a2_b504_3ded06e75f0c/fhirStores/nodejs_docs_samples_test_fhir_storeb5946f8c_2ffa_4fad_ade8_61803a3fcc4e
```

That resource doesn't look right to me. Is is supposed to have `nodej-docs-samples_test_uuid` *twice* in the path?

**Pull request merged**